### ISSUE TITLE
UTM does not make newly created VMs appear immediately, displays "The file couldn't be opened" upon changing settings

### ISSUE THREAD

**Issue by KendrickLamar2003** [utmapp/UTM#6398]
**Describe the issue**  
So I am trying to make a Win95 VM, and after going through the wizard, it does not appear in the list, it only does show up after relaunching the app. Whenever I want to change something about the VM, upon pressing "Save...", it hits me with an error that simply says: "The file couldn't be opened" and keeps me in the settings menu. But the changes I make ARE saved. Worst of all, doesn't allow me to mount an ISO to a VM. This still happens even after deleting all files relating to UTM + the app itself.
**Configuration**  
* UTM Version: 4.5.3
* macOS Version: 14.5 (23F79)
* Mac Chip (Intel, M1, ...): M1 8-core GPU

**Comment by osy** [comment]
Sorry for the late response but can you provide a screen recording? I cannot reproduce the issue.

**Comment by naveenrajm7** [comment]
Hello, 
I ran into a similar issue but in a different way. Can be reproduced

I am automating a few things to add to my plugin.
* I create a VM through the script.
* Trying changing settings manually 
* I will be thrown a "The file couldn't be opened" error.
* But the settings I added will be saved.

So behaviour is same as the OP observed.

Steps to reproduce.

1. create_vm.applescript
```applescript
tell application "UTM"
    --- specify a boot ISO
    set iso to POSIX file "ubuntu-24.04-live-server-arm64.iso"
    --- create a new QEMU VM for ARM64 with a single 64GiB drive
    set vm to make new virtual machine with properties {backend:qemu, configuration:{name:"QEMU ARM64", architecture:"aarch64", drives:{{removable:true, source:iso}, {guest size:65536}}}}
end tell
```

2. Execute
```bash
osascript create_vm.applescript
```

3. In UI trying changing any settings
![Screenshot 2024-08-19 at 6 36 31 PM](https://github.com/user-attachments/assets/b229d897-0c9c-420c-ba48-c6470e711d75)


The iso file is present , can actually run the VM if I didn't touch settings. (but I need display to observe, so went to settings)
